src/background.js

```diff
@@ -77,8 +100,24 @@ function checkDevinStatus() {
     const messageText =
       statusBarMessage.textContent || statusBarMessage.innerText;
     if (messageText.startsWith("Devin is awaiting")) {
-      return "Devin is waiting";
+      // Extract the last message time from the status bar
+      const lastMessageTime = statusBarMessage.querySelector(
+        "[data-state='closed']",
+      ).textContent;
```

Comment:
I want to reconsider the how to get latest message time. Can we get the value something like this.

1. Get all history item
2. Get latest history item (= last of item)
3. Get time

```
const lastMessageTime = document.querySelectorAll(".message-history .message-history--item")
```